### PR TITLE
Use shared Travis configuration to fix the CI

### PR DESCRIPTION
**Pull request by edwintorok**
We are using OCaml 4.08.1 now, lets control this from a central place in
xs-opam.

### CODE REVIEW

**Comment by lippirk** [comment]
this doesn't match the example what we have in xs-opam

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+  - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
+  - wget https://raw.githubusercontent.com/xapi-project/xs-opam/master/tools/xs-opam-ci.env
+  - source xs-opam-ci.env
 script: bash -ex .travis-docker.sh
 env:
   global:
-    - OCAML_VERSION="4.07"
-    - DISTRO="debian-unstable"
     - PINS="xapi-squeezed:."
     - PACKAGE="xapi-squeezed"
```
Comment (lippirk) [inline .travis.yml]: in the example we have `PACKAGE` inside a job:

```
  jobs:
    - PACKAGE="EXAMPLE"
```

Comment (edwintorok) [reply .travis.yml]: Didn't realize we had an example (just copied one from xapi-idl), updated now, lets see if it works.

Comment (psafont) [reply .travis.yml]: The jobs directive is only relevant when there are multiple packages and we want to test each one separately.

**Pull request merged**